src/core/messages/DraftBotErrorEmbed.ts

```diff
@@ -1,15 +1,17 @@
 import {DraftBotEmbed} from "./DraftBotEmbed";
-import {User} from "discord.js";
+import {CommandInteraction, User} from "discord.js";
 import {Translations} from "../Translations";
 
 /**
  * Default error embed with the title and description formatted. If you just want the red color, see {@link DraftBotEmbed#setErrorColor}
  */
 export class DraftBotErrorEmbed extends DraftBotEmbed {
-	constructor(user: User, language: string, reason: string, isCancelling = false) {
+	constructor(user: User, interaction: CommandInteraction, language: string, reason: string, isCancelling = false) {
```

Comment:
mwais, la ca va être dur de faire descendre le nombre de params à 4

Mon idée (pas pour tout de suite, mais faudra y penser) faire un embed spécial pour le cancelling (en gros, on fait un embed qui étend celui ci, le constructeur c'est super(<les paramètes de celui ci>) avec un this.formatAuthor qui viendra réécrire l'ancien. par définition ce seront les mêmes embeds en terme de style, ils seront juste créés de facon différente niveau code